testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -4076,3 +4077,439 @@ QUnit.module('Bands', moduleConfig, () => {
         });
     });
 });
+
+QUnit.module('customizeCell', moduleConfig, () => {
+    QUnit.test('Header cells, modifying predefined font settings', function(assert) {
```

Comment:
Here we test that customizeCell is called for each item of dxDataGrid: header, band, data cell, totals, group header, group header totals, group footer totals, ...more?
I suggest to change test names: one for each context.
And, check that args members return correct values in each context: notNull(args.gridCell), equal(args.gridCell.rowType, 'header'), equal(args.gridCell.data, 'header'), notNull(args.pdfCell), ... more?

Reply:
Ok, I'll rework these tests according these recommended improvements.